File: VB/Q303243/MyGridView.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Text
Imports DevExpress.XtraGrid.Views.Grid

Namespace Q303243
	Public Class MyGridView
		Inherits GridView
		Public Overrides Function IsExistAnyRowFooterCell(ByVal rowHandle As Integer) As Boolean
			Return True
		End Function
	End Class
End Namespace

File: CS/Q303243/MyGridView.cs
using System;
using System.Collections.Generic;
using System.Text;
using DevExpress.XtraGrid.Views.Grid;

namespace Q303243 {
    public class MyGridView : GridView {
        public override bool IsExistAnyRowFooterCell(int rowHandle) {
            return true;
        }
    }
}
